src/chigame/users/views.py

```diff
@@ -120,3 +120,31 @@ def cancel_friend_invitation(request, pk):
     else:
         messages.error(request, "Something went wrong please try again later!")
     return redirect(reverse("users:user-profile", kwargs={"pk": request.user.pk}))
+
+
+@login_required
+def remove_friend(request, pk):
+    sender = User.objects.get(pk=request.user.id)
+    receiver = User.objects.get(pk=pk)
+    sender_profile = sender.profile
+    sender_profile.friends.remove(receiver)
+    messages.success(request, "Friend removed successfully.")
+    return redirect(reverse("users:user-profile", kwargs={"pk": request.user.pk}))
+
+@login_required
+def invite_to_game(request, pk, match_id):
+    sender = User.objects.get(pk=request.user.id)
+    receiver = User.objects.get(pk=pk)
+    match = User.objects.get(pk=match_id)  
```

Comment:
when defining match, I think you defined it incorrectly, and it should be Match.objects instead of User.objects, as the User model doesn't have a match_id